CMakeLists.txt

```diff
@@ -6,11 +6,13 @@ catkin_simple()
 
 include(ExternalProject)
 
+set(YAML_CPP_TAG 11607eb5bf1258641d80f7051e7cf09e317b4746)
```

Comment:
Unfortunately all releases seem to be not using C++11 features. So they still contain std::auto_ptr which creates a warning. According to this [link](https://github.com/jbeder/yaml-cpp/releases/tag/yaml-cpp-0.5.3) the next release should have this fixed. But for now I would suggest to leave it. Wdyt?